src/annotator/config/index.js

```diff
@@ -31,6 +31,9 @@ export default function configFrom(window_) {
     }),
     query: settings.query,
     requestConfigFromFrame: settings.hostPageSetting('requestConfigFromFrame'),
+    configFrameAncestorLevel: settings.hostPageSetting(
+      'configFrameAncestorLevel'
+    ),
```

Comment:
I think it would be preferable to enable `requestConfigFromFrame` to be an object with `origin` and `ancestorLevel` properties OR a string. My rationale for this is because it is easier to communicate in documentation that there is an optional field which, if specified, is an object with two required properties, rather than two separate optional fields where, if one is specified, the other should be as well.

Note that this is separate from the change to use two separate query params in Via to specify the parent frame. In that context I think using separate params is reasonable since there isn't a "natural" way to encode nesting in query strings.